clusterloader2/README.md

```diff
@@ -48,12 +53,23 @@ Two always available parameters are ```{{.Name}}``` and ```{{.Index}}```
 which specifies object name and object replica index respectively. \
 Example of a template can be found here: [load rc template].
 
+### Overrides
+
+Overrides allow to inject new variables values to the template. \
+Many tests define input parameters. Input parameter is a variable
+that potentially will be provided by the test framework. Cause input parameters are optional,
+each reference should be opaqued with ```DefaultParam``` function that will
+handle case if given variable doesn't exist. \
+Example of overrides can be found here: [overrides]
+
 ## Measurement
 
 Currently available measurements are:
 - **APIResponsiveness** \
 This measurement creates summary for latency and number for server api calls.
- Api calls are divided by resource, subresource, verb and scope.
+Api calls are divided by resource, subresource, verb and scope. \
+This measurement is a SLO which means if [api call latency restrictions] are violated,
```

Comment:
This measurement verifies if API call latencies SLO is satisfied.

Reply:
Done.